keps/sig-node/3983-drop-in-configuration/README.md

```diff
@@ -77,10 +77,10 @@ Components in Kubernetes can similarly be configured by multiple entities and pr
 
 ### Goals
 
-* Add support for a "--config-dir" flag to the kubelet to allow users to specify a drop-in directory, which will override the configuration for the Kubelet located at `/etc/kubernetes/kubelet.conf`
+* Add support for a `--config-dir` flag to the kubelet to allow users to specify a drop-in directory, which will override the configuration for the Kubelet located at `/etc/kubernetes/kubelet.conf`
 * Extend kubelet configuration parsing code to handle files in the drop-in directory.
 * Define Kubernetes best-practices for configuration definitions, similarly to [API conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md). This is intended for other maintainers who would wish to setup a configuration object that works well with drop-in directories.
-* As a goal for beta, Add ability to easily view the effective configuration that is being used by kubelet.
+* In the beta phase, we plan to add end-to-end (E2E) tests for the feature, ensuring its reliability and functionality. We are going to remove the `KUBELET_CONFIG_DROPIN_DIR_ALPHA` environment variable, which was introduced during the alpha phase for testing purposes. The feature will remain off by default during the beta phase, allowing users to enable it explicitly when needed.
```

Comment:
what does it mean to keep it off by default? The only config is `--config-dir` flag

Reply:
I'd remove this note completely, just say that env variable is gone

Reply:
ok, let me fix the wording